### PR TITLE
ci: add MADlib to extension job

### PR DESCRIPTION
**Pull request by kalensk**
ci: add MADlib to extension job

Pipeline: https://cm.ci.gpdb.pivotal.io/teams/main/pipelines/gpupgrade:finalUpgradeMadlibExtensionJob
(Pipeline is still running...If needed, we will fix any small tweaks necessary.)

### CODE REVIEW

File: ci/scripts/load-extensions.bash [hunk 1]
```diff
@@ -19,28 +20,38 @@ time ssh -n mdw "
...
+    echo 'Installing MADlib...'
+    gppkg -i /tmp/madlib_source.gppkg
+    /usr/local/greenplum-db-source/madlib/bin/madpack -p greenplum -c /postgres install
+    psql -d postgres <<SQL_EOF
+        DROP TABLE IF EXISTS madlib_test_type;
+        CREATE TABLE madlib_test_type(id int, value madlib.svec);
+        INSERT INTO madlib_test_type VALUES(1, '{1,2,3}'::float8[]::madlib.svec);
+        INSERT INTO madlib_test_type VALUES(2, '{4,5,6}'::float8[]::madlib.svec);
+        INSERT INTO madlib_test_type VALUES(3, '{7,8,9}'::float8[]::madlib.svec);
+        CREATE VIEW madlib_test_view AS SELECT madlib.normal_quantile(0.5, 0, 1);
```
Comment (soumyadeep2007) [inline ci/scripts/load-extensions.bash:43]: We currently don't run any sort of validation on these on the target cluster. I think we should perform a `\d+` on the table and the view along with a `select *` on the view on both source and target clusters.

**Review by soumyadeep2007 — COMMENTED** [review]
Unless this PR blocks other work, we can hold off on merging this until the madlib release is done.

File: ci/scripts/load-extensions.bash (hunk 1 above)
Comment (kalensk) [reply ci/scripts/load-extensions.bash:43]: We are in the process of investigating using the isolation2 framework to test these.

**Comment by kalensk** [comment]
> Unless this PR blocks other work, we can hold off on merging this until the madlib release is done. 

That will take at least 2-4 weeks. There is no need to hold off on this PR just to update the bucket. This also lets gpupgrade test the MADlib changes end-to-end before releasing MADlib.

**Review by soumyadeep2007 — APPROVED** [review]

**Pull request merged**